ci/run_task.sh

```diff
@@ -146,7 +143,7 @@ do_test() {
     $cargo build
     $cargo test
 
-    if [ -n "${EXAMPLES+x}" ]; then
+    if [ -n "${EXAMPLES}" ]; then
```

Comment:
In 6646692a37c06388b58c8177d7ee5add1bce313a:

What were these `+x`s supposed to mean? AFAIK they just ensure that the `-n` check will always pass and the `-z` checks never will.

Reply:
Correct, and also I was annoyed at myself for writing syntax that I had to look  up when I tried to read it. Hence I added the more brain-dead set the vars to `""` in the loop.